Fix function internetConnection to prevent run ping and internet connection error.
https://github.com/steineggerlab/ufcg/blob/cf9cfe3c2950c8d17447864da4574014db5a90f7/src/module/DownloadModule.java#L121
--- src/module/DownloadModule.java.orig	2025-01-28 18:06:36.553823000 +0100
+++ src/module/DownloadModule.java	2025-01-28 18:28:36.023487000 +0100
@@ -119,9 +119,7 @@
     }
 
     private boolean internetConnection(){
-        String[] raw = Shell.exec(String.format("ping -c 1 -W %d ufcg.steineggerlab.workers.dev", GenericConfig.OS.equals("Linux") ? 1 : 1000), true, 0);
-        if(raw == null || raw.length < 2) return false;
-        return raw[raw.length - 2].contains(GenericConfig.OS.equals("Linux") ? "1 received" : "1 packets received");
+        return true;
     }
     private void printManual() {
         System.out.println(ANSIHandler.wrapper(" UFCG - download", 'G'));
